core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -221,6 +221,9 @@ var ExplorationEditorPage = function() {
       saveDraftButton, 'Save Draft button is not clickable');
     await saveDraftButton.click();
 
+    await waitFor.invisibilityOf(
+      element(by.css('.protractor-test-saving-changes-dots')),
```

Comment:
nit: can we use a global variable for the selector instead (i.e. define this in the top of the file)?